mappings/net/minecraft/client/render/BackgroundRenderer.mapping

```diff
@@ -6,6 +6,9 @@ CLASS net/minecraft/class_758 net/minecraft/client/render/BackgroundRenderer
 	FIELD field_4041 nextWaterFogColor I
 	FIELD field_4042 lastWaterFogColorUpdateTime J
 	METHOD method_3210 render (Lnet/minecraft/class_4184;FLnet/minecraft/class_638;IF)V
+		ARG 0 camera
+		ARG 1 tickDelta
+		ARG 2 world
```

Comment:
`world` for a float?

Reply:
This may be a static method, or `ARG 0` won't be namable

Reply:
yep - `public static void render(Camera camera, float tickDelta, ClientWorld world, int integer, float float5)`